scene/resources/particles_material.cpp

```diff
@@ -233,48 +233,48 @@ void ParticlesMaterial::_update_shader() {
 	code += "uniform vec3 gravity;\n";
 
 	if (color_ramp.is_valid()) {
-		code += "uniform sampler2D color_ramp;\n";
+		code += "uniform sampler2D color_ramp: repeat_disable;\n";
```

Comment:
Just a syntactic note: I've typically seen shader hints written like this:

```glsl
uniform sampler2D color_ramp : repeat_disable;
```

As opposed to this:

```glsl
uniform sampler2D color_ramp: repeat_disable;
```

The [Godot shader language style guide](https://docs.godotengine.org/en/latest/tutorials/shaders/shaders_style_guide.html) doesn't make a recommendation on this yet, but we should figure one out.

Reply:
The example in the shading language docs does it this way:

```gdscript
shader_type spatial;

uniform vec4 color : hint_color;
uniform float amount : hint_range(0, 1);
uniform vec4 other_color : hint_color
```

Reply:
I don't have an opinion either way. I updated it to have the space like:
```glsl
uniform sampler2D color_ramp : repeat_disable;
```